__tests__/hooks/useAuth.test.tsx

```
@@ -0,0 +1,452 @@
+import React from 'react';
+import { render, screen, waitFor, act } from '@testing-library/react';
```

ESLint

[error] 'act' is defined but never used.

(@typescript-eslint/no-unused-vars)